Core/src/Seeding/EstimateTrackParamsFromSeed.cpp

```diff
@@ -107,10 +107,14 @@ Acts::BoundMatrix Acts::estimateTrackParamCovariance(
       // note that we rely on the fact that sigma theta is already computed
       double varianceTheta = result(eBoundTheta, eBoundTheta);
 
-      // transverse momentum contribution
+      // contribution from sigma(q/pt)
+      variance += std::pow(
+          config.initialSigmaQoverPt * std::sin(params[eBoundTheta]), 2);
+
+      // contribution from sigma(pt)/pt
       variance += std::pow(config.initialSigmaPtRel * params[eBoundQOverP], 2);
```

Comment:
Sorry for the naive question, but can you explain why are you still adding this term if you have a non 0 initial sigmaQoverPt here? 
if I write q/p as (q/pT)*sin(theta), then 
sigma(q/p)^2 = |sin(theta)|^2*sigma(q/pT)^2 + |(q/p)*1/tan(theta)|^2*var(theta). Why there is an additional pT term?

Reply:
we still want to keep the error contribution from $\sigma(p_T)/p_T$ which scales with the initial $q/p$. the additional term with $\sigma(q/p_T)$ basically only sets a minimal error for high momentum.

Reply:
I think it does not hurt to add both. I do not see an immediate use case but the user is in control of these values and can choose how to set them. Potentially it might be useful to parameterize with all three values?

Reply:
I find it a bit confusing having an initial q/p and an initial q/pT, honestly. If you pass both != 0, then you basically double count the error. It doesn't necessarily harm, but perhaps one could add some comments / make it clear in the config or in the header. 

If you pass q/pT = 0, then we are in the original behavior, if q/p=0 instead, you use the new formulation. if both q/pT and q/p !=0 then one inflates the original error in the end, I guess.

Reply:
I will try to clarify in the documentation 👍 

Right, if you set both != 0 you get a mixture. At the end you just have more free params at hand to model something. I thought it might be useful and it is less confusing without a p vs pT toggle